<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Note App</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QListWidget" name="list_notes"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="button_newNote">
          <property name="text">
           <string>Yeni</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_editNote">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Düzenle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_deleteNote">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Sil</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="button_lock">
          <property name="text">
           <string>Kilitle</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>672</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDosya">
    <property name="title">
     <string>Dosya</string>
    </property>
    <addaction name="actionNewNote"/>
    <addaction name="actionEditNote"/>
    <addaction name="actionDeleteNote"/>
    <addaction name="separator"/>
    <addaction name="actionLock"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuDosya"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewNote">
   <property name="text">
    <string>Yeni</string>
   </property>
  </action>
  <action name="actionEditNote">
   <property name="text">
    <string>Düzenle</string>
   </property>
  </action>
  <action name="actionDeleteNote">
   <property name="text">
    <string>Sil</string>
   </property>
  </action>
  <action name="actionLock">
   <property name="text">
    <string>Kilitle</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Çıkış</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionLock</sender>
   <signal>triggered()</signal>
   <receiver>button_lock</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>611</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNewNote</sender>
   <signal>triggered()</signal>
   <receiver>button_newNote</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEditNote</sender>
   <signal>triggered()</signal>
   <receiver>button_editNote</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDeleteNote</sender>
   <signal>triggered()</signal>
   <receiver>button_deleteNote</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
